yt_dlp/extractor/npo.py

```diff
@@ -115,7 +113,8 @@ class NPOIE(NPOBaseIE):
         },
         'params': {
             'skip_download': True,
-        }
+        },
+        'skip': 'Currently broken',
```

Comment:
```suggestion
        },
```
What's the point of tests if you skip when it breaks?

Reply:
I reverted the change. The problem seems to be that the token and embed code generated by the npostart API don't work for some videos. Does this need to be addressed, or is it fine to keep the test failing for now?

Reply:
It would be great if you can address it, but I'll merge the PR even without it

Reply:
I have pushed a potential solution. I am unsure if this is acceptable. If not, I would just revert the last commit.